form.html

```diff
@@ -40,13 +44,14 @@ <h1>What's the occassion?</h1>
             <section id="generateActivity">
                 <h1>Don't know?<br> Let us choose an activity for you!</h1>
                 <!-- Link onclick event when function is created in js-->
-                <button id="boredAPI" onclick="getActivity()">Generate Activity</button>
+                <button class="startButton" onclick="getActivity()">Generate Activity</button>
```

Comment:
If you are adding the class for css purposes just add it into the tag that way we still have our id's available to us in the JS